posthog/api/error_tracking.py

```diff
@@ -115,10 +115,11 @@ def retrieve(self, request, *args, **kwargs):
             record = fingerprint_queryset.filter(fingerprint=fingerprint).first()
 
             if record:
-                if not record.issue_id == self.request.GET.get("pk"):
+                if not str(record.issue_id) == self.kwargs.get("pk"):
                     return JsonResponse({"issue_id": record.issue_id}, status=status.HTTP_308_PERMANENT_REDIRECT)
 
-                serializer = self.get_serializer(record.issue)
+                issue_with_first_seen = ErrorTrackingIssue.objects.with_first_seen().get(id=record.issue_id)
```

Comment:
this was my best attempt at ensuring `first_seen` was included for the serializer - not sure if there'd be a better way to do this!